### ISSUE TITLE
Feature request: Allow to move the window behind the Finder menu on MAC OS

### ISSUE THREAD

**Issue by jmeile** [justinfrankel/licecap#17]
Hi

I want to record an action that requires me to select an option from the Finder-Menu, the menu on the top of the desktop (on this case: Go > Utilities). I can only position the Licecap Window above the menu, so, that menu won't be recorded. It would be nice if you included a way of positioning the window behind that menu.

There is a similar App to yours that allows this: Screen-Cast-O-Matic, so, this is possible. You can move the windows behind that menu. I think that the window type must be different. They use some kind of transparent frame. Perhaps there is some possibility to specify windows types on the code.

Thanks

Best regards
Josef

**Comment by VictorBjelkholm** [comment]
Yes! I would want this too, I wanted to record the URL-bar of Safari, but can't because of the top border in LICEcap or the finder menu in OSX.

Screenshot:
<img width="879" alt="screen shot 2016-06-29 at 15 23 54" src="https://cloud.githubusercontent.com/assets/459764/16453561/83935c18-3e0d-11e6-9911-9675a812b393.png">
